Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/RobotCameraPipeline.java

```diff
@@ -61,24 +70,30 @@ public Mat processFrame(Mat input) {
                 }
             }
 
-            // get the bounding rectangle around the largest contour
-            Rect boundingRect = Imgproc.boundingRect(contours.get(maxValIdx));
+            // crops contour if the area is below a certain size
+            if (maxVal >= Constants.CONTOUR_MIN_SIZE)
+            {
+                // get the bounding rectangle around the largest contour
+                Rect boundingRect = Imgproc.boundingRect(contours.get(maxValIdx));
 
-            // get moments
-            Moments moments = Imgproc.moments(contours.get(maxValIdx), false);
+                // get moments
+                Moments moments = Imgproc.moments(contours.get(maxValIdx), false);
 
-            // draw the bounding rectangle on the frame
-            Imgproc.rectangle(input, boundingRect, new Scalar(0, 255, 0), 10);
+                // draw the bounding rectangle on the frame
+                Imgproc.rectangle(input, boundingRect, new Scalar(0, 255, 0), 10);
 
-            if (moments.get_m00() > 0) {
-                xPosition = boundingRect.x + (boundingRect.width * 0.5);
-                yPosition = boundingRect.y + (boundingRect.height * 0.5);
-                width = boundingRect.width;
+                if (moments.get_m00() > 0)
+                {
```

Comment:
Use the autoformat to fix all these types of formatting issues. https://google.github.io/styleguide/javaguide.html#s4.1-braces
```suggestion
                if (moments.get_m00() > 0) {
```